src/track/heaptrack.sh.cmake

```diff
@@ -100,11 +106,21 @@ while true; do
                 echo "Missing output argument."
                 exit 1
             fi
-            output=$(readlink -f $2)
+            output=$2
+            if [ -n "$(echo $output | grep '%h')" ]; then
```

Comment:
the `grep` isn't needed, is it? you could just replace via `sed` directly, dito below

Reply:
Yes, you are correct, the conditional can be removed.